src/crimson/os/seastore/transaction.h

```diff
@@ -82,15 +82,15 @@ class Transaction {
   };
   get_extent_ret get_extent(paddr_t addr, CachedExtentRef *out) {
     LOG_PREFIX(Transaction::get_extent);
-    if (retired_set.count(addr)) {
-      return get_extent_ret::RETIRED;
-    } else if (auto iter = write_set.find_offset(addr);
+    if (auto iter = write_set.find_offset(addr);
 	iter != write_set.end()) {
       if (out)
 	*out = CachedExtentRef(&*iter);
       SUBTRACET(seastore_cache, "{} is present in write_set -- {}",
                 *this, addr, *iter);
       return get_extent_ret::PRESENT;
+    } else if (retired_set.count(addr)) {
+      return get_extent_ret::RETIRED;
```

Comment:
Why change the order of the retired_set and write_set checks?

Reply:
absolute paddr in write_set also appears in retired_set.

remove extent at laddr 16k
map existing 4k extent at laddr 16k (its paddr is same with original extent's)